superset-frontend/src/components/Popover/index.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Popover } from 'src/common/components';
+
+// Eventually Popover can be wrapped and customized in this file
+// for now we're just redirecting
+export default Popover;
```

Comment:
Depending on how we feel about the appearance of the popover, we may not _need_ to do this, _but_ I like this scaffolding to set us up for a little Emotion styling here. If the day comes that we _do_ tweak the styles, let's just be sure to add it to Storybook!

Reply:
Yeah I didn't know if we wanted to do this redirect here. Main goal was to fix `SpatialControl`

Reply:
I think it's a good pattern to do this, actually.